README.md

```diff
@@ -1,3 +1,24 @@
+* **Restart Policy Max Attempts**: This check will issue a warning if you specify a restart policy on the service which has a `max_attempts` with no `window`.  If you set a `max_attempts` on your restart policy it will only allow the replica for a service to restart that many times before it will stop trying to restart it.  If you do not also configure a `window` alongside this Swarm will look at any exited replicas when determining if it has restarted too many times, even if those replicas exited days or weeks before.  The `window` configuration will ensure that the replica has restarted too many times in a fixed window so the `max_attempts` will work as intended.  To remove this warning make sure you always specify a `window` in your restart policy if you use `max_attempts`.  See the [restart policy configuration](https://docs.docker.com/compose/compose-file/#restart_policy) for more details.
+* **Restart Policy Delay**: This check will issue a warning if you are not using `max_attempts` in your service's restart policy and have not configured a `delay`.  Services without a restart delay that are having issues starting will constantly spawn new replicas as they error out which can put strain on the swarm's managers and routing layers.  Adding a delay gives a bit more breathing room since the service will not thrash while it is having an issue.  There are two ways to fix this warning.  You can add a [restart policy delay configuration](https://docs.docker.com/compose/compose-file/#restart_policy) or you can add a `max_attempts` configuration which will put an upper bound on the number of times the service will be restarted if there are issues.  See the information on the `Restart Policy Max Attempts` if you go with `max_attempts` since it documents some gotchas with that approach.
+
+## Running the script
+
+When executed the script takes an optional argument which is the name of a single service to check.  If no arguments are provided it will instead check every service it can find.  By default it will only log a summary of each service along with details of the checks that have warnings or failures.  The verbose option (`-v` or `--verbose`) may be provided and the script will log details about each check that succeeds for each service.  The quiet option (`--quiet`) will only log information for services that have failures.  The exit code of the script indicates the number of total failures that were found so it can be used to fail a CI plan.
```

Comment:
Doesn't this require a UCP bundle? Is there anything about that?

Reply:
OK, I see you can run it a number of ways, but still could be documented

Reply:
Yes, I am planning to document that in a separate PR.  I want to put this into a Docker container and document how to run it with that.  That is why I don't include documentation on how to run it and the dependencies involved since they will be different once there is a Docker container.

Reply:
Makes sense. Thanks.